.gitignore

```diff
@@ -36,3 +36,6 @@ src/cloud-api-adaptor/*.qcow2
 .git-commit
 .git-commit.tmp
 *.tar.gz
+
+# Secrets
+install/overlays/gcp/GCP_CREDENTIALS
```

Comment:
Adding the comment on the commit message here.  You can remove the statement on GKE now since 1909 is merged.

Reply:
Are you proposing we bring GKE support back?

If it's okay with you, can we proceed with this version to establish basic GCP support quickly? I can add GKE in a follow-up PR?

Reply:
I don't see anything which is specific to GKE and the same code will work for GKE (similar to what we have for aws/eks and azure/aks). Hence suggested to drop the statement from the commit message.
Or for GKE there is a need for separate code?

Reply:
Ohh understood now, fixed the commit message. thanks. :)